joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/frontend/SymbolTable.scala

```diff
@@ -25,7 +25,7 @@ abstract class SBKey(val identifier: String) {
 object SBKey {
   protected val logger: Logger = LoggerFactory.getLogger(getClass)
   def fromNodeToLocalKey(node: AstNode): Option[LocalKey] = {
-    Option(node match {
+    val option = Option(node match {
```

Comment:
This seems unnecessary?

Reply:
Yes, that was for debugging, forgot to remove, will remove